<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1513</width>
    <height>939</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="SImageList" name="image_list">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>500</y>
      <width>231</width>
      <height>381</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="SModelList" name="model_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>891</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="SLabel3DAnnotation" name="vtk_panel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>871</width>
      <height>841</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1513</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_Load_Scenes"/>
    <addaction name="action_Save_Scenes"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Load_Scenes">
   <property name="text">
    <string>&amp;Load Scenes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save_Scenes">
   <property name="text">
    <string>&amp;Save Scenes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SImageList</class>
   <extends>QFrame</extends>
   <header location="global">simagelist</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SModelList</class>
   <extends>QFrame</extends>
   <header location="global">smodellist</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SLabel3DAnnotation</class>
   <extends>QFrame</extends>
   <header location="global">slabel3dannotation</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
